MTM-54655 - changelog - fixed alarm count getting out of sync (#2420)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-536-0-fixed-alarm-count-out-of-sync.md

---

> date: 2024-09-05

Reviewer: ```suggestion
date: 2024-09-11
```

> title: Fixed alarm count getting out of sync

Reviewer: ```suggestion
title: Alarms counts in managed objects in sync with alarm counts in the database
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-54655
version: 10.20.536.0
---

> Fixed an issue with the alarm count in managed object getting out of sync with the actual count of alarms in the database.

Reviewer: ```suggestion
The alarm count in managed objects could get out of sync with the actual count of alarms in the database. This issue has been fixed and alarms counts in managed objects are now in sync with alarm counts in the database.
```
